src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -971,8 +971,8 @@ window.qBittorrent.DynamicTable = (function() {
                         img_path = "images/stalledDL.svg";
                         break;
                     case "pausedDL":
-                        state = "torrent-stop";
-                        img_path = "images/torrent-stop.svg";
+                        state = "stopped";
```

Comment:
Isn't previous state name mentioned anywhere else in the code?

Reply:
I had done a "find" text for `torrent-stop`. All the other hits seem to refer to the svg name/path.

For what it's worth, I have loaded the WebUI and loaded a torrent. I paused/resumed it. There's nothing immediately broken as far as I can tell.

Reply:
Hm... For what purposes is "state" used, since changing it does not affect anything? Maybe something did go unnoticed... Isn't it dangerous to merge such changes immediately before the release?

Reply:
OK, I reverted the change regarding the `state` variable.
I observed that it changes the tooltip when hovering over the icon column in the transfer list.